.pnp.js

```diff
@@ -3937,7 +3937,9 @@ function $$SETUP_STATE(hydrateRuntimeState, basePath) {
             ["@types/node", "npm:14.14.31"],
```

Comment:
what tool is generating this file?

Also, can we remove all .zip files introduced in this PR (or maybe the whole .yarn folder)?

Reply:
`.php.js` and `.yarn` are generated by yarn2: they replace `node_modules` in a much more reliable way and since the cache is compressed, it is good practice to commit it so that yarn install doesn't need to download dependencies.